app/build.gradle

```diff
@@ -33,6 +33,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    namespace 'com.chaquo.python.console'
```

Comment:
This is an important setting, so it should appear at the top of the `android` block, which is where the Android Studio new project wizard would put it.

Reply:
Done.